cfgov/jinja2/v1/_includes/molecules/image-text-50-50.html

```diff
@@ -11,23 +11,27 @@
 
    See [GHE]/flapjack/Modules-V1/wiki/50-50-Image-&-Text
 
-   settings:              An object with the following options for settings.
-   settings.title:        A string containing title of module
-   settings.image_path:   A string containing the url for the image
-   settings.image_alt:    A string containing alt text for screenreaders
+   settings:               An object with the following options for settings.
```

Comment:
Should we make the settings documentation use the same format as the ` image_text_25_75` macro?